taqo_client/lib/widgets/app_drawer.dart

```diff
@@ -68,7 +68,22 @@ class _TaqoAppDrawerWidget extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final authProvider = Provider.of<AuthProvider>(context);
-
+    final ThemeData theme = Theme.of(context);
+    final TextStyle textStyle = theme.textTheme.bodyText2;
+    final List<Widget> aboutTaqoWidget = <Widget>[
```

Comment:
Please remove the type annotations (between `final` and the variable names) on these lines